### ISSUE TITLE
Please add a release/changelog

### ISSUE THREAD

**Issue by groob** [jessepeterson/margarita#33]
Hi, 

I maintain the macadmins/reposado docker container, which bundles margarita as one of the options. 
It' would be helpful if we could have a release version/git tag to pin/correlate builds with.

**Comment by jessepeterson** [comment]
Good idea. I'll try to find some time to learn about Github releases, tagging, and release notes.
